OSDOCS-16409: Cluster Admin CQAs (#116629)
openshift/openshift-docs: modules/dedicated-aws-vpn-creating-customer-gw.adoc

// Module included in the following assemblies:
//
// * rosa_cluster_admin/cloud_infrastructure_access/dedicated-aws-vpn.adoc

:_mod-docs-content-type: PROCEDURE
[id="dedicated-aws-vpn-creating-customer-gw_{context}"]
= Creating a customer gateway and virtual private gateway

[role="_abstract"]
Create a customer gateway and, if needed, a virtual private gateway to prepare for a VPN connection to your {product-title} cluster.

.Prerequisites

* Hardware VPN gateway device model and software version, for example Cisco ASA running version 8.3. See the Amazon VPC link:https://docs.aws.amazon.com/vpc/latest/adminguide/Introduction.html#DevicesTested[Network Administrator Guide] to confirm whether your gateway device is supported by AWS.
* Public, static IP address for the VPN gateway device.

> * BGP or static routing: if BGP, the ASN is required. If static routing, you must configure at least one static route.

Reviewer: ```suggestion
* Border Gateway Protocol (BGP) or static routing: if using (or if you choose) BGP, the Autonomous System Number (ASN) is required. If static routing, you must configure at least one static route.
```

* *Optional*: IP and Port/Protocol of a reachable service to test the VPN connection.

.Procedure

. Log in to the {product-title} AWS Account Dashboard, and navigate to the VPC Dashboard.

> . Under *Virtual private cloud* click on *Your VPCs* and identify the name and VPC ID for the VPC containing the {product-title} cluster.

Reviewer: ```suggestion
. Under *Virtual private cloud* click *Your VPCs* and identify the name and VPC ID for the VPC containing the {product-title} cluster.
```

. Under *Virtual private network (VPN)* click *Customer gateways*.

> . Click *Create customer gateway* and give it a meaningful name.

Reviewer: ```suggestion
. Log in to the {product-title} AWS Account Dashboard.
. Navigate to the VPC Dashboard.
. Under *Virtual private cloud* click on *Your VPCs* and identify the name and VPC ID for the VPC containing the {product-title} cluster.
. Under *Virtual private network (VPN)* click *Customer gateways*.
. Click *Create customer gateway* and name it.
```

. Enter the ASN of your customer gateway device in the *BGP ASN* field.

> . Enter the IP address for your customer gateway devices's external interface in the *IP address* field.

Reviewer: ```suggestion
. In the *IP address* field, enter the IP address for your customer gateway devices' external interface.
OR
. In the *IP address* field, enter the IP address of an external interface for your customer gateway devices.
```

devices' or devices (no possessive at all).

. Click *Create customer gateway*.
. If you do not already have a Virtual Private Gateway attached to the intended VPC:

> .. From the VPC Dashboard, click on *Virtual Private Gateways*.

Reviewer: ```suggestion
.. From the VPC Dashboard, click *Virtual Private Gateways*.
```
